db/sync.go

```diff
@@ -0,0 +1,236 @@
+package db
+
+import (
+	"database/sql"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const lastModifiedField = "last_modified"
+
+func Sync(sourceDB, targetDB *sql.DB) error {
+	tables, err := getTablesWithLastModifiedFrom(sourceDB)
+	if err != nil {
+		log.WithError(err).Error("Error in getting list of tables with last_modified field from the source DB")
+		return err
+	}
+
+	for _, table := range tables {
```

Comment:
Minor opportunity to reduce time;

We have 10 connections per DB (source / target), assuming that each table copy operation does not contain any foreign keys/constraints, we can run the sync operation in parallel up to 10 go routines. Though, if introducing parallelism complicates the testing in any way, we can skip it.

Reply:
According to the disaster recovery process, we expect that the source database contains few records. So, I agree that introducing parallelism improve performances, but it introduces more complexity for debugging and testing. In my opinion, for the amount of data that potentially needs to be transfered, it does not make sense to introduce such complexity.

Reply:
👍